[2024-01-23T18:41:46.439+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2024-01-23T18:41:46.469+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2024-01-23T18:41:46.470+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-23T18:41:46.909+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2022-10-03 00:00:00+00:00
[2024-01-23T18:41:46.934+0000] {standard_task_runner.py:60} INFO - Started process 4178 to run task
[2024-01-23T18:41:46.999+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2022-10-03T00:00:00+00:00', '--job-id', '514', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmprwlh8zcs']
[2024-01-23T18:41:47.038+0000] {standard_task_runner.py:88} INFO - Job 514: Subtask get_stackoverflow_data
[2024-01-23T18:41:47.765+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-10-03T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-23T18:41:48.410+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2022-10-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-03T00:00:00+00:00'
[2024-01-23T18:41:49.804+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T18:41:49.919+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20221003T000000, start_date=20240123T184146, end_date=20240123T184149
[2024-01-23T18:41:50.620+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T18:41:51.181+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
